augur/tasks/git/facade_tasks.py

```diff
@@ -194,8 +187,8 @@ def update_analysis_log(repos_id,status):
 
 
 
-        for commit in trimmed_commits:
```

Comment:
Here down is the factored out trim_commits method, correct, @IsaacMilarky ... trying to make sure I understand what's happening.